pages/linux/pngcheck.md

```diff
@@ -0,0 +1,21 @@
+# pngcheck
+
+> Forensics tool for validating the integrity of PNG based (`.png`, `.jng`, `.mng`) image files.
+> Can also extract embedded images and text from a file.
+> More information: <http://www.libpng.org/pub/png/apps/pngcheck.html>.
+
+- Verify the integrity of an image file:
+
+`pngcheck {{path/to/file.png}}`
+
+- Check the file with [v]erbose and [c]olorized output:
+
+`pngcheck -vf {{path/to/file.png}}`
```

Comment:
![\[c\]](https://user-images.githubusercontent.com/35269695/196065072-4b89b0a8-7633-4604-8b7c-87a09d1378f9.png)
![-vf](https://user-images.githubusercontent.com/35269695/196065095-f55bfff9-716b-40d6-b46e-058b8ce2fdad.png)

Am I missing something?

Reply:
Fixed in commit 
e9e28b0

Thank you